guide/message_transformation/message_transformation.bal

```diff
@@ -0,0 +1,370 @@
+import ballerina/http;
```

Comment:
Shall we add the licence header to all files?

Reply:
Done